core/src/main/java/org/stellar/anchor/util/Log.java

```diff
@@ -69,6 +72,7 @@ public static void debugF(final String format, final Object... args) {
   public static void error(final String msg) {
     Logger logger = getLogger();
     logger.error(msg);
+    Metrics.counter("logger", "type", "error").increment();
```

Comment:
What is the value of counting the amount of error/warning logs? I'm not opposed to this, just trying to understand the reason behind it.

Reply:
seems like something an operator would want to keep tabs on and maybe even alert on and look into if there are a high number of error/warnings.